models/comments.model.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../db/connection')
+
+
+const checkExists = (id) => {
+    return db.query(
+        `SELECT * FROM reviews WHERE review_id = $1;`, [id]
+    ).then((result) => {
+    if (result.rowCount === 0) {
+        return Promise.reject({ status: 404, msg: 'Id does not exist'})
+    }   
+})
+}
```

Comment:
Could you reuse a previous function you have made for this?

Reply:
Thanks! Oh yes, of course! I'll change so that it uses the fetchReview function instead.